doc/Security.xml

```diff
@@ -947,13 +987,290 @@ Added certificate-based client authentication</revremark>
         </orderedlist>
       </section>
     </section>
+    <section xml:id="_Ref395181339">
+      <title>JWT-based client authorization</title>
+      <para>Unlike with HTTP Digest and WS-UsernameToken authentication, this specification defines how to associate clients with the different User Levels as defined in the ONVIF Core Specification. Devices are not expected to have the user credentials stored in their memory, instead they will verify that the user has been correctly authenticated by an external service based on OpenID Connect and authorize accessing different functions, based on the user level presented in the supplied JWT embedded in the binary security token.</para>
+      <para>The content of the binary security token is the base64-encoded JWT according to [RFC 7519].</para>
+      <para>The JOSE header shall declare that the encoded object is a JWT, and the JWT is a JWS that is signed using ECDSA using P-256 (secp256r1) curve and SHA-256 hash algorithm. Additionally, it must include the <emphasis>kid</emphasis> claim as defined by [RFC 7515], which indicates the ID associated to the key used to sign the JWT. If the absolute path of the JWK Set URL is different from https://iss/.well-known/jwks.json, the header may include an optional <emphasis>jku</emphasis> claim.</para>
+      <para>The JWT shall include the following standard claims:</para>
+      <para>
+        <itemizedlist>
+          <listitem>
+            <para><emphasis>iss</emphasis>: Issuer, the URI of the server that generated the JWT.</para>
+          </listitem>
+          <listitem>
+            <para><emphasis>aud</emphasis>: Audience, a list of strings representing the recipients that the JWT is intended for. Each principal intended to process the JWT shall identify itself with a value in the audience claim. If the principal processing the claim does not identify itself with a value in the audience claim when this claim is present, then the JWT shall be rejected.</para>
+          </listitem>
+          <listitem>
+            <para><emphasis>exp</emphasis>: Expiration Time.</para>
+          </listitem>
+          <listitem>
+            <para><emphasis>nbf</emphasis>: Not before.</para>
+          </listitem>
+        </itemizedlist>
+      </para>
+      <para>For the <emphasis>exp</emphasis>, <emphasis>nbf</emphasis> claims, a device shall reject a token when the current time is not within the range of claims nbf and exp . It shall reject a token when at least one of the claims is missing.</para>
```

Comment:
Should we really mandate that a device rejects a token without an nbf claim if it is optional in the JWT spec?